docs/operation.md

```diff
@@ -64,7 +64,7 @@ Verify that your node sees the same longest blockchain as the testnet.
 
 Inspect the current top of the blockchain as seen by the testnet:
 ```bash
-curl http://31.13.249.70:3013/v2/top
```

Comment:
Is `||` mean to be or on exit status?
* Not sure about exit status of curl (e.g. no `-f`) in all error conditions
* User may copy paste and not understand what is going on

May I suggest replacing `||` with `##` i.e. comment? Though change optional.

Reply:
> Is || mean to be or on exit status?

yes

I see that `curl` have sophisticated error/exit codes https://linux.die.net/man/1/curl
This makes me thing that it exit with 0 really only on success, I think it will cover 99% is the cases, while we're looking really for simple cases where the IP is available but not HTTP listener on it, or some HTTP error code during node boot phase.


> May I suggest replacing || with ## i.e. comment? Though change optional.

I could, but I'm not sure what's better. As you wrote users are copy paste and go on, I guess most of the users don't care (and don't understand) what's going on as long as it works.

I'm concerned that additional comment with additional command will confuse them more ? 🤔

Reply:
You convinced me. Please leave.